src/classes/apiCache/index.ts

```diff
@@ -57,15 +61,21 @@ export class ApiCache {
    * @param {number} durationInMS Cache expiration in ms
    * @returns Promise<string>
```

Comment:
boolean?

Reply:
No, `redis.get()` returns the exact REDIS output, which is `OK`. We keep getting this wrong in other places.

Reply:
My remark was just to say that the method signature seems different that the jsdoc:

```ts
async setCache(req: Request, data: unknown, durationInMS: number = this.config.expirationInMS): Promise<boolean> {
```

```js
  /**
   * @description Stores data into cache.
   * @param {Request} req Express request associated with the data
   * @param {unknown} data - Data to cache
   * @param {number} durationInMS Cache expiration in ms
   * @returns Promise<string>
   */
```

Reply:
Oh got it, i'll fix it